src/components/PopUpWithConfirm.js

```diff
@@ -0,0 +1,29 @@
+import PopUp from "./Popup.js";
+
+export default class PopUpWithConfirm extends PopUp{
+    constructor(popupSelector) {
+        super({popupSelector});
+        this._popupForm = this._popupElement.querySelector(".modal__form");
+        this._cardElement = document.querySelector(".card");
+}
+
+close() {
+    this._popupForm.reset();
+    super.close();
```

Comment:
`this._popupForm.reset();` is not needed in `PopUpWithConfirm` (you don't have any inputs here)

Reply:
That makes sense. Just removed